src/configuration.cpp

```diff
@@ -193,13 +193,13 @@ ssr::conf_struct ssr::configuration(int& argc, char* argv[])
   // load system-wide config file (Linux et al.)
   load_config_file("/etc/ssr.conf",conf);
   // load user config file (Mac)
-  std::string filename = getenv("HOME");
-  filename += "/Library/SoundScapeRenderer/ssr.conf";
-  load_config_file(filename.c_str(),conf);
+  fs::path home_dir = pathtools::get_home_dir();
```

Comment:
This could be `path()`. Would that really make sense?

I was expecting the config file only to be loaded if there actually is a HOME directory. If not, nothing should happen, right?

Now it seems like `/Library/SoundScapeRenderer/ssr.conf` is loaded but that has just been loaded before (in line 192), so loading it again doesn't really make sense?

I guess you should check `if (home_dir != fs::path())` or something?